src/main/java/base/Figure.java

```diff
@@ -0,0 +1,19 @@
+package base;
+
+public class Figure implements AreaCalculable, Drawable {
+    protected String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
```

Comment:
The `getArea` method should not have an implementation in the `Figure` class if it's meant to be overridden in every subclass. Consider making it abstract to force subclasses to provide their own implementation.